src/style.css

```diff
@@ -1 +1,90 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+/*test shows 6-2. please help to realize my mistakes*/
+
+html {
+  font-family: "Roboto", Arial, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+body {
+  margin: 0;
+}
+  
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  width: 100%;
+  padding: 0 50px;
+}
+  
+.nav_list {
+  display: flex;
+  text-align: center;
+  margin: 0;
+  padding: 0;
+}
+
+.links {
+  margin-right: 20px;
+  padding: 23px 0;
+  position: relative;
+  list-style: none;
+  justify-content: center;
+  text-align: center;
+}
+
+.links:last-child {
+  margin-right: 0;
+}
+
+.link_a {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  cursor: pointer;
+  font-family: inherit;
+  display: block; /*??????????*/
+}
+
+.logo {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+}
+ 
+.is-active,
+.link_a:hover {
+  color: #00acdc;
+}
+  
+.is-active::after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  background: #00acdc;
+  border-radius: 8px;
+}
+
+.link_a:active::after {
+  content: "";
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  height: 4px;
+  width: 100%;
+  background-color: #00acdc;
+  /*border-radius: 8px;*/
+}
```

Comment:
I understand the reason you've added it, but you could delete it here since we can't select some tab permanently for now